commit 45864e49a05aa446b9d99c464c1a9f46956ed32c
Author: Cezary Rojewski <cezary.rojewski@intel.com>
Date:   Fri Mar 11 16:35:42 2022 +0100

    ASoC: Intel: avs: Implement CLDMA transfer
    
    SKL and KBL rely on a dedicated HDAudio DMA stream for code loading and
    authentication. The implementation of this specific mechanism for
    SKL-based platforms re-uses HDAudio DMA (streaming) functions found in
    HDA library to avoid duplication of functionality.
    
    Signed-off-by: Amadeusz Sławiński <amadeuszx.slawinski@linux.intel.com>
    Signed-off-by: Cezary Rojewski <cezary.rojewski@intel.com>
    Link: https://lore.kernel.org/r/20220311153544.136854-16-cezary.rojewski@intel.com
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/sound/soc/intel/avs/registers.h b/sound/soc/intel/avs/registers.h
index 73699d46821b..3fd02389ed2b 100644
--- a/sound/soc/intel/avs/registers.h
+++ b/sound/soc/intel/avs/registers.h
@@ -22,7 +22,9 @@
 #define AVS_ADSP_REG_ADSPIS		(AVS_ADSP_GEN_BASE + 0x0C)
 
 #define AVS_ADSP_ADSPIC_IPC		BIT(0)
+#define AVS_ADSP_ADSPIC_CLDMA		BIT(1)
 #define AVS_ADSP_ADSPIS_IPC		BIT(0)
+#define AVS_ADSP_ADSPIS_CLDMA		BIT(1)
 
 #define AVS_ADSPCS_CRST_MASK(cm)	(cm)
 #define AVS_ADSPCS_CSTALL_MASK(cm)	((cm) << 8)